sdk/data/aztables/README.md

```diff
@@ -84,39 +123,74 @@ connection string to the client's `NewServiceClientFromConnectionString` method.
 
 ##### Creating the client from a SAS token
 To use a [shared access signature (SAS) token][azure_sas_token], provide the token as a string. If your account URL includes the SAS token, omit the credential parameter. You can generate a SAS token from the Azure Portal under [Shared access signature](https://docs.microsoft.com/rest/api/storageservices/create-service-sas) or use the `ServiceClient.GetAccountSASToken` or `Client.GetTableSASToken()` methods.
 
 ```golang
-cred, err := aztables.NewSharedKeyCredential("myAccountName", "myAccountKey")
-handle(err)
-service, err := aztables.NewServiceClient("https://<myAccountName>.table.core.windows.net", cred, nil)
-
-resources := aztables.AccountSASResourceTypes{Service: true}
-permission := aztables.AccountSASPermissions{Read: true}
-start := time.Now()
-expiry := start.AddDate(1, 0, 0)
-sasUrl, err := service.GetAccountSASToken(resources, permission, start, expiry)
-handle(err)
-
-sasService, err := aztables.NewServiceClient(sasUrl, azcore.AnonymousCredential(), nil)
-handle(err)
+import (
+    "fmt"
+    "time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/data/aztables"
+)
+
+func main() {
+	cred, err := aztables.NewSharedKeyCredential("myAccountName", "myAccountKey")
+	if err != nil {
+		panic(err)
+	}
+	service, err := aztables.NewServiceClientWithSharedKey("https://<myAccountName>.table.core.windows.net", cred, nil)
+
+	resources := aztables.AccountSASResourceTypes{Service: true}
+	permission := aztables.AccountSASPermissions{Read: true}
+	start := time.Now()
+	expiry := start.AddDate(1, 0, 0)
+	sasURL, err := service.GetAccountSASToken(resources, permission, start, expiry)
+	if err != nil {
+		panic(err)
+	}
+
+	serviceURL := fmt.Sprintf("https://<myAccountName>.table.core.windows.net/?%s", sasURL)
+	sasService, err := aztables.NewServiceClientWithNoCredential(serviceURL, nil)
+	if err != nil {
+		panic(err)
+	}
+}
 ```
 
 ##### Creating the client for Azurite
 If you are using the [Azurite](https://github.com/Azure/Azurite) emulator you can authenticate a client with the default connection string:
-```golang
-connStr := "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;"
-svc, err := NewServiceClientFromConnectionString(connStr, nil)
-handle(err)
+```go
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/data/aztables"
+)
+
+func main() {
+    connStr := "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;"
+    svc, err := NewServiceClientFromConnectionString(connStr, nil)
+    if err != nil {
+        panic(err)
+    }
 
-client, err := svc.CreateTable(context.TODO(), "AzuriteTable", nil)
-handle(err)
+    client, err := svc.CreateTable(context.TODO(), "AzuriteTable", nil)
```

Comment:
Is `AzuriteTable` a typo? Is it supposed to be `AzureTable`?

Reply:
This sample is specifically for connecting to the Azurite emulator so I went with the name to match.